inspirehep/modules/theme/templates/inspirehep_theme/format/record/authors/Author_HTML_detailed.html

```diff
@@ -34,7 +34,7 @@
       <div class="row">
         <div class="col-md-12 summary" id="detailed-header-top">
           <div class="col-md-2">
-            <img class="profile-image img-circle" src="{{ record['positions'][0]['email'] | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com' width="100%">
+            <img class="profile-image img-circle" src="{{ record['positions'][0].get('email', 'gravatar@inspirehep.net') | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com' width="100%">
```

Comment:
Right. I guess I can amend this commit to exclude this modification (and also remove the commit that adds `DEBUG = True` so that the build does not fail).